diagnostic.js

```diff
@@ -29,26 +29,79 @@ mongoose.Promise = global.Promise;
 mongoose.connect('mongodb://localhost/mongoose-crud');
 
 /// ADD YOUR CODE BELOW
+const db = mongoose.connection;
 
-const create = (name, description, startYear, endYear) => {};
+const Movement = require('./models/movement.js');
+
+const done = function() {
+  db.close();
+};
```

Comment:
Not necessary here - this was part of the UI functionality of the command line app. We needed to ensure that the Mongo connection was closed each time the script finished running so that there weren't lots of open connections at the end of the day. Here, we (almost certainly) want to keep the connection open.